pkg/ipcache/metadata.go

```diff
@@ -511,6 +538,32 @@ func (ipc *IPCache) resolveIdentity(ctx context.Context, prefix netip.Prefix, in
 	return id, isNew, err
 }
 
+// updateReservedHostLabels adds or removes labels that apply to the local host.
+// The `reserved:host` identity is special: the numeric identity is fixed
+// and the set of labels is mutable. (The datapath requires this.) So,
+// we need to determine all prefixes that have the `reserved:host` label and
+// capture their labels. Then, we must aggregate *all* labels from all IPs and
+// insert them as the `reserved:host` identity labels.
```

Comment:
Just clarifying a bit more.

```suggestion
// capture their labels. Then, we must aggregate *all* labels from all prefixes and
// update the labels that correspond to the `reserved:host` identity.
```

Reply:
Additionally, I think it would be useful to call out is that this function is doing another merging of metadata of which there are two now: one in this function and the other is the main ipcache metadata consolidator. The main difference from the latter is that we have to consider other prefixes, whereas the latter is per prefix.

Reply:
OK, I added some wording to that effect.